test/karma/test-app/non-shadow-to-shadow-slot-relocation/karma.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { setupDomTests } from '../util';
+
+describe('non-shadow-to-shadow-slot-relocation', () => {
+  const { setupDom, tearDownDom } = setupDomTests(document);
+  let app: HTMLElement | undefined;
+  let host: HTMLElement | undefined;
+
+  beforeEach(async () => {
+    app = await setupDom('non-shadow-to-shadow-slot-relocation/index.html');
+    host = app.querySelector('non-shadow-host');
+  });
+
+  afterEach(tearDownDom);
+
+  it('should correctly render slotted content', () => {
+    expect(host).toBeDefined();
+
+    const mainDiv = host.firstElementChild.firstElementChild;
+    expect(mainDiv).toBeDefined();
+    expect(mainDiv.children.length).toBe(2);
+
+    // Main content
+    expect(mainDiv.children[0].tagName).toBe('B');
```

Comment:
Non-blocking: For some of these assertions, it might be nice to pull some of the repeated accessors (e.g. `mainDiv.children[1]`) into a variable for the sake of 'cleaning' the code up. But how far we take that is pretty subjective - up to you how much you wanna do (or not)